src/index.html

```diff
@@ -458,6 +459,11 @@ <h3 class="gallery__heading">Łukasz Nowacki</h3>
     </p>
   </div>
 </footer>
+<!-- Inline for faster parsing -->
+<script>
+  document.querySelector('body').classList.toggle('with-js');
+  document.querySelector('.main-header').classList.toggle('main-header--sticky');
```

Comment:
It's not faster whatsoever in this case. Both calls will take less than 1ms so **it doesn't matter**.

jsperf results are not really meaningful for real-life development and IMHO shouldn't be used for anything else than just experiments. For example, this jsperf test case tests calling querySelector over a milion times while on the website we need to do it only once.

Thanks for your input, but I must say that trying to optimise things which are already optimal and are not causing any performance troubles is simply a waste of your precious time.